src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs

```diff
@@ -357,7 +357,7 @@ public bool AttachDebuggerToProcess(int pid, CancellationToken cancellationToken
             var ackPayload = _dataSerializer.DeserializePayload<EditorAttachDebuggerAckPayload>(ackMessage);
             if (!ackPayload.Attached)
             {
-                EqtTrace.Warning(ackPayload.ErrorMessage);
+                EqtTrace.Warning($"DesignModeClient.AttachDebuggerToProcess: Attaching to process failed: {ackPayload.ErrorMessage}");
```

Comment:
This throws debug assert if the ErrorMessage is null. Super if this breaks in your tests.